com.unity.formats.fbx/Editor/ExportModelSettings.cs

```diff
@@ -95,6 +95,13 @@ public override void OnInspectorGUI ()
             exportSettings.SetExportUnredererd(EditorGUILayout.Toggle(exportSettings.ExportUnrendered));
             EditorGUI.EndDisabledGroup ();
             GUILayout.EndHorizontal ();
+            
+            GUILayout.BeginHorizontal();
+            EditorGUILayout.LabelField(new GUIContent("Preserve Import Settings for Existing FBX",
+                "If checked, the import settings from the overwritten FBX will be carried over to the new version."), GUILayout.Width(LabelWidth - FieldOffset));
+            exportSettings.SetPreserveImportSettings(EditorGUILayout.Toggle(exportSettings.PreserveImportSettings));
+            EditorGUI.EndDisabledGroup();
```

Comment:
What is this `EndDisabledGroup` matching against? 
I see you added a `BeginHorizontal` and `EndHorizontal` scope, but perhaps this one should be removed?

One nice way to organize these "scopes" is to add an indent to the code between them